src/test/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+
+profile.active=test
```

Comment:
Use Spring profiles?

Reply:
No.We need to have different constructors for the AmazonSQS client based on the enviroment, so we decided to keep the same name as spring to define it in the properties. Do you prefer that i change the name in something else like "enviroment"?